server/integration/jgroups/src/main/java/org/infinispan/server/jgroups/ManagedSocketFactory.java

```diff
@@ -80,7 +80,7 @@ private Socket register(final Socket socket) {
 
     @Override
     public ServerSocket createServerSocket(String name) throws IOException {
-        return this.register(this.factory.createServerSocket(name));
+        return this.factory.createServerSocket(name);
```

Comment:
@tristantarrant @pferraro I'm not sure if this is the right way to fix this, but if we try to register an unbound socket we get an `IllegalStateException`. It worked with JGroups 3.6.x worked because `JChannel.startStack()` was overwriting the top protocol's socket factory with its own `DefaultSocketFactory`.